scripts/ci-entrypoint.sh

```diff
@@ -125,6 +128,8 @@ wait_for_nodes() {
     # Ensure that all nodes are registered with the API server before checking for readiness
     local total_nodes="$((CONTROL_PLANE_MACHINE_COUNT + WORKER_MACHINE_COUNT + WINDOWS_WORKER_MACHINE_COUNT))"
     while [[ $("${KUBECTL}" get nodes -ojson | jq '.items | length') -ne "${total_nodes}" ]]; do
+        "${KUBECTL}" get nodes -owide
```

Comment:
Wouldn't this be quite noisy in the output?

Reply:
Yes, for most of the time, the output is not necessary. However, my intention is to help debug some situation like this:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-cloud-provider-azure-e2e-ccm-capz/1532236185013850112
I cannot repro in local setup.

Reply:
I moved them to on_exit()